requirements/default.txt

```diff
@@ -3,7 +3,7 @@ numpy>=1.10.0
 qtpy>=1.7.0
 scipy>=1.2.0
 scikit-image>=0.15.0
-vispy>=0.5.3
+vispy==0.5.3
```

Comment:
yikes. I guess we don't really have a choice right now due to all of the mixing of stuff, but the ideal workflow would be:
1. implement the changes in the `vispy` main repo
2. backport it to be compatible with the `0.5.3` code
3. monkeypatch it in `napari` iff `vispy==0.5.3`